schema/types.go

```diff
@@ -77,3 +78,12 @@ type PodInfo struct {
 type ContainerInfo struct {
 	InfoType
 }
+
+// Supported metric names, used as keys for all map[string]*store.TimeStore
+const cpuLimit = "cpu/limit"
```

Comment:
That is the eventual goal, however there are a few issues to consider, such as:
- time resolution of limits vs time resolution of usage.
- mismatch of limit/usage timestamps.
- input resolution versus output resolution for historical data.
- instantaneous derived stats (e.g. current utilization percentage) and the required conversions of limits/usage values.

For those reasons, I believe it's better if we address the format of the resource struct together with the design of the derived stats.